app/events/admin.go

```diff
@@ -108,7 +108,8 @@ func (a *admin) MsgHandler(update tbapi.Update) error {
 
 	// make a message with spam info and send to admin chat
 	spamInfo := []string{}
-	resp := a.bot.OnMessage(bot.Message{Text: update.Message.Text, From: bot.User{ID: info.UserID}})
+	// check only, don't update the storage
```

Comment:
Same thing as in the Request structure, a comment like "check only, don't whitelist the user based on the admin response because in situation X it would lead to whitelisting unexpected user because of Y" would be more useful.